examples/bert/yamls/main/hf-bert-base-uncased.yaml

```diff
@@ -19,6 +19,13 @@ model:
   use_pretrained: false # Train the model from scratch. Set to true to start from the HF off-the-shelf weights.
   pretrained_model_name: *tokenizer_name
   tokenizer_name: *tokenizer_name
+  # This implementation generally uses the default architecture values for from the Hugging Face BertConfig object
+  # These values can be changed here:
+  model_config:
```

Comment:
For the HF example, which comes with a `use_pretrained` option, we may want to clarify that config changes that change the size of the model (compared to the defaults registered for `pretrained_model_name`) will create issues when using `use_pretrained: true`.

Reply:
Agreed that this is worth including

Reply:
I've added an extra line in the descriptions
```
# These values can be changed here when pretraining from scratch. Note that these should only be used
  # if used_pretained: false, otherwise the model will not be loaded properly
  ```